### PR TITLE
Disable tests on windows/mono - green build

### PR DESCRIPTION
**Pull request by radical**

### CODE REVIEW

**Comment by cdmihai** [comment]
LGTM. Please open an issue about investigating / fixing the tests marked with mono-windows-failing

File: dir.targets [hunk 1]
```diff
@@ -32,7 +32,7 @@
 
   <!-- Respect the traits set up by PlatformSpecific attribute. -->
   <PropertyGroup>
-    <XunitOptions Condition="'$(OS)'=='Windows_NT'">$(XunitOptions) -notrait category=nonwindowstests</XunitOptions>
+    <XunitOptions Condition="'$(OS)'=='Windows_NT'">$(XunitOptions) -notrait category=nonwindowstests -notrait category=mono-windows-failing</XunitOptions>
```
Comment (rainersigwald) [inline dir.targets]: Should this be conditional on `MonoBuild==true` analagously to `mono-osx-failing` below?

Comment (radical) [inline dir.targets]: Yikes! Fixing this and another issue :/

**Comment by radical** [comment]
Updated and added some cases that I had missed.

**Comment by AndyGerlicher** [comment]
LGTM

**Pull request merged**